### PR TITLE
Feature/failed notification admin

### PR DESCRIPTION
**Pull request by stevenbal**
issue https://github.com/open-zaak/open-zaak/issues/310

### CODE REVIEW

File: src/openzaak/components/zaken/api/viewsets.py [hunk 1]
```diff
@@ -429,7 +434,9 @@ class ZaakObjectViewSet(
 
 
 class ZaakInformatieObjectViewSet(
+    FailedNotificationMixin,
     NotificationCreateMixin,
+    NotificationDestroyMixin,
```
Comment (stevenbal) [inline src/openzaak/components/zaken/api/viewsets.py]: @sergei-maertens this one was missing here, but not on the viewset for `BesluitInformatieObject`, so I added it here as well

File: src/openzaak/notificaties/mixins.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+
+class FailedNotificationMixin:
+    def notify(self, status_code, data, instance=None):
+        stream = StringIO()
+        console = logging.StreamHandler(stream=stream)
+        logger = logging.getLogger("vng_api_common.notifications.viewsets")
+        logger.addHandler(console)
+
+        logger.addFilter(FailedNotificationFilter())
```
Comment (joeribekker) [inline src/openzaak/notificaties/mixins.py]: This doesn't seem like the right approach to me.

Why not create a log handler that puts stuff in a model? And register it via settings?

This whole mixin concept seems inappropriate.

Let's discuss with @sergei-maertens

Comment (sergei-maertens) [reply src/openzaak/notificaties/mixins.py]: Indeed, I had something like https://pypi.org/project/django-db-logger/ in mind instead

**Comment by stevenbal** [comment]
closed in favor of https://github.com/open-zaak/open-zaak/pull/354

**Pull request closed without merging**